validation-test/Reflection/reflect_UInt16.swift

```diff
@@ -4,7 +4,7 @@
 
 // RUN: %target-run %target-swift-reflection-test %t/reflect_UInt16 | %FileCheck %s --check-prefix=CHECK-%target-ptrsize
 
-// REQUIRES: objc_interop
+// REQUIRES: basic_reflection_support
```

Comment:
Should the condition be called `reflection_test_support` or something more specific?

Reply:
@vedantk what do you think? I'm ok with either.

Reply:
'reflection_test_support' sounds better than 'basic_reflection_support' to me, since it's not clear the basic/non-basic distinction is.